.github/workflows/c-bindings.yml

```diff
@@ -0,0 +1,35 @@
+name: C bindings tests CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+  merge_group:
+    branches: [ "main" ]
+
+jobs:
+  c-bindings:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - uses: dtolnay/rust-toolchain@stable
+
+    - name: Install Protoc
+      uses: arduino/setup-protoc@v2
+
+    - name: Configure libsql-sqlite3
+      working-directory: libsql-sqlite3
+      run: CFLAGS="-DSQLITE_ENABLE_COLUMN_METADATA=1" ./configure --enable-all --enable-releasemode
+
+    - name: Build libsql-sqlite3
+      working-directory: libsql-sqlite3
+      run: make
```

Comment:
We can drop these two lines and then add [this](https://github.com/tursodatabase/libsql/blob/main/.github/workflows/rust.yml#L31) before cargo build so it can cache.

Reply:
Updated. :)